packages/front-end/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
+    "downlevelIteration": true,
```

Comment:
`const {users} = useUser()` is of type `Map<Key, Value>` which is non-iteratable without this compiler flag. In order to get all the users and make dropdown options, I needed to iterate over them.

Reply:
This option adds a lot of extra code to the js bundle.  I would rather just use forEach to iterate over maps:

```ts
users.forEach((user) => {
  
})
```